example/extending/newlang/5-hello-python/build.mill

```diff
@@ -0,0 +1,118 @@
+// using the offical https://mypy.readthedocs.io/en/stable[Documentation] for `mypy` 
+
+// === Defining our Sources
+
+// Next, We define the `sources` task to help gather source files for processing. 
+// `sources` sets a base directory for Python source files and enable easy access to Python files in a project,
+// making it straightforward to retrieve all relevant code files for tasks like building, testing, or analysis.
+
+def sources: T[PathRef] = Task.Source(millSourcePath / "src")
+
+// === Type Checking
+
+// The `mainFileName` Task provides the name for the main python script file for running and 
+// The `typeCheck` Task  ensures that the python setup is set and Checks the main file for error free code.
+// This task is considered as a part of code which is responsible for Error Checking of the Main Script 
+// and setting up setup for python script runable environment.
+
+def mainFileName: T[String] = Task { "main.py" }
+def typeCheck: T[PathRef] = Task {
+  val pythonVenv = setup().path
+
+  os.call(
+    (pythonVenv,"-m","mypy","--strict",sources().path / mainFileName()),
+    stdout = os.Inherit
+  )
+
+  PathRef(pythonVenv)
+}
+
+// When `typeCheck` task sets up the Setup for Virtual Environment and checks the main file for error free code 
+// then we have `main.py` and running this file will result in either a generic greeting 
+// or a personalized one based on the command-line arguments passed, allowing for flexible interaction with the user.
+
+/** See Also: src/main.py */
+
+
+// === Running
+
+// The `run` function executes the main Python file (`main.py`) and accepts command-line arguments from the user. 
+// It uses `os.call` to run the Python interpreter(from Virtual Environment) with the main file's path 
+// and any additional arguments. The output is shown in the console, allowing for real-time interaction.
+  
+def run(args: mill.define.Args) = Task.Command {
+  val pythonVenv = typeCheck().path
+  
+  os.call(
+    (pythonVenv,sources().path / mainFileName(),args.value),
+    stdout = os.Inherit
+  )
+}
+
+// Note that we use `stdout = os.Inherit` since we want to display any output to the user,
+// rather than capturing it for use in our command.
+
+// ```graphviz
+// digraph G {
+//   rankdir=LR
+//   node [shape=box width=0 height=0 style=filled fillcolor=white]
+//   setup -> typeCheck -> run
+//   sources -> typeCheck
+//   mainFileName -> typeCheck
+//   sources -> run
+//   mainFileName -> run
+//   mainFileName [color=green, penwidth=3]
+//   run [color=green, penwidth=3]
+// }
+// ```
+
+// Running run command will return the result to the console.
+
+/** Usage
+
+> ./mill typeCheck
+Success: no issues found in 1 source file
+
+> ./mill run Mill Python
+Hello, Mill Python!
+
+*/
+
+// Uncomment the Result variable line(Line Number:13) in the `src/main.py` code to see `typeCheck` error
```

Comment:
actually sir i want to show error also but the problem is that if i write code for error then it will fail in ci test as this will me a major error from mypy side so how can i do this 

please Guide me a little Bit Sir...